### DISCUSSION TITLE
Inference with YOLOR

### DISCUSSION THREAD

**Discussion by asluborski98** [obss/sahi#428] [Q&A] [answered]
Hello,

I was wondering if it is possible to do inference on YOLOR? 

there is no pip install so I was wondering how we can load framework in? 

Thanks.

**Comment by fcakyon** [comment, marked as answer]
Hello @asluborski98! Since there is no pip package for YOLOR we cannot add official support into sahi package but you definitely can implement a version that works in your local.

You can follow the steps given in contribution section: https://github.com/obss/sahi#contributing and you can take the custom DETIC implementation as reference: https://github.com/machinelearnear/large-scale-object-detection-with-sahi-detectron2/blob/main/quick-start.ipynb

**Reply by kadirnar** [reply]
@akashAD98  look here
